doc/manpages/bob-build-dev.rst

```diff
@@ -86,6 +86,12 @@ Options
 ``-f, --force``
     Force execution of all build steps
 
+``-i, --installshared``
+    Install shared packages to a given location (has to be combined with --shared)
```

Comment:
I think it would be OK to install them by default, but I want to control if it should be installed or not. Would you go with a "dont install relocatable packages" parameter?

Reply:
Wouldn't you just *not* define the shared location? Or set it to an empty string in your `user.yaml` if you defined it globally? I don't think this is something that should be a command line option because you don't typically change your mind between invocations.

Do I miss something?

Reply:
I meant to use them but not to install. Set the variable to an empty string would cause not to use it either. --shared isn't necessary but something like "--dontinstallsharedpackages" ;) would be nice

Reply:
I think the problem is that the shared packages are re-installed every time. This is not necessary and we should change the "install" part to only do it if the package is not yet available in the shared location. Then you only have the overhead of copying it once.